From ed8271e4738edf10c218b5a5289ec9ab0dcf7303 Mon Sep 17 00:00:00 2001
From: Chanho Park <chanho61.park@samsung.com>
Date: Thu, 26 Sep 2013 14:35:25 +0900
Subject: [PATCH 0806/1302] usb: gadget: slp: remove rndis_init

This patch removes rndis_init because it was already enabled during module_init.

Signed-off-by: Chanho Park <chanho61.park@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 drivers/usb/gadget/slp.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/usb/gadget/slp.c b/drivers/usb/gadget/slp.c
index 6a22cf9..0a5fec4 100644
--- a/drivers/usb/gadget/slp.c
+++ b/drivers/usb/gadget/slp.c
@@ -477,14 +477,6 @@ static int rndis_function_bind_config(struct slp_multi_usb_function *f,
 		rndis_control_intf.bInterfaceProtocol = 0x03;
 	}
 
-	/* ... and setup RNDIS itself */
-	ret = rndis_init();
-	if (ret < 0) {
-		dev_err(f->dev, "rndis_init failed(ret:%d)\n", ret);
-		gether_cleanup(rndis->edev);
-		return ret;
-	}
-
 	/* Android team reset "rndis_string_defs[0].id" when RNDIS unbinded
 	 * in f_rndis.c but, that makes failure of rndis_bind_config() by
 	 * the overflow of "next_string_id" value in usb_string_id().
-- 
1.8.3.2

